packages/messages/src/browser/notifications-contribution.ts

```diff
@@ -51,12 +51,19 @@ export class NotificationsContribution implements FrontendApplicationContributio
             text: this.getStatusBarItemText(count),
             alignment: StatusBarAlignment.RIGHT,
             priority: -900,
-            command: NotificationsCommands.TOGGLE.id
+            command: NotificationsCommands.TOGGLE.id,
+            tooltip: this.getStatusBarItemTooltip(count)
         });
     }
     protected getStatusBarItemText(count: number): string {
         return `$(bell) ${count ? ` ${count}` : '' }`;
     }
+    protected getStatusBarItemTooltip(count: number): string {
+        if (this.manager.centerVisible) {
+            return 'Hide Notifications';
+        }
+        return `${count ? `${count} Notifications` : 'No Notifications'}`;
```

Comment:
nit: @vince-fugnitto could you make it singular for a single item, please?

Reply:
Sure :) I updated the pull request

Reply:
thanks!